my-app/src/app/features/courses-page/courses-page.component.html

```diff
@@ -15,12 +15,20 @@
       <span>Add course</span>
     </button>
   </div>
-  <div class="courses-page__items-container">
-    <cp-courses-item *ngFor="let course of courses" [course]="course"
+  <div class="courses-page__items-container" *ngIf="this.courses && this.courses.length; else noResults">
```

Comment:
1. No need to use `this` in a template context
2. `courses?.length` would be enough